### DISCUSSION TITLE
Improve image cache-control header when using minimumCacheTTL

### DISCUSSION THREAD

**Discussion by iscekic** [vercel/next.js#35660] [Ideas]
### Describe the feature you'd like to request

By default, next returns `cache-control: public, max-age=0, must-revalidate` for non-immutable images.
My expectation when using `minimumCacheTTL` is for the header to be `cache-control: public, max-age=${Math.max(minimumCacheTTL, originMaxAge)}, must-revalidate`.

This should help efficiency:
* reduce the amount of `304` responses / etag checks
* setups where there is a single cache / load balancer in front of multiple next instances would use less cpu (no need to generate an image multiple times if it was already generated by a different instance)

### Describe the solution you'd like

When `minimumCacheTTL` is present, the headers next sends get set to `cache-control: public, max-age=${Math.max(minimumCacheTTL, originMaxAge)}, must-revalidate`.

Alternatively, a flag in the settings which forces this behavior (eg. `forceExplicitCacheHeaders` or similar).

### Describe alternatives you've considered

Using `headers` to do this "manually" - however [this is advised against by the docs](https://nextjs.org/docs/api-reference/next/image#minimum-cache-ttl=).

**Comment by balazsorban44** [comment]
I think https://github.com/vercel/next.js/pull/35596 is related

**Reply by iscekic** [reply]
Not the exact behavior I was aiming for, but the spirit is the same - reduce load on the server by utilizing browser/cdn caches more effectively.
